packages/api/src/schema/schema.yaml

```diff
@@ -486,6 +486,10 @@ components:
           example: D8321C3E-B29C-45EB-A1BB-A623D8BE0F65
           description:
             ID of object store where to which this stream was recorded
+        recordingSessionId:
+          type: string
```

Comment:
Are we going to be querying by these fields? If so, you probably want to add an `index: true` directive to this (or even `unique: true` where that's relevant (e.g. sessions table))

Reply:
Added index here https://github.com/livepeer/studio/pull/1692/commits/918b9f73b5a7fcc663a27eea37fd6748dd5cdc27

I'm not sure about `unique`, didn't add it for now. My reasoning is that, even though I expect them to be unique, it's not a requirement from the studio perspective.